### ISSUE TITLE
Build error with latest async-trait

### ISSUE THREAD

**Issue by kvinwang** [lawliet89/rocket_cors#111]
--> /home/kvin/.cargo/registry/src/github.com-1ecc6299db9ec823/rocket_cors-0.6.0-alpha1/src/fairing.rs:29:18
   |
29 |         let _ = &__arg2;
   |                  ^^^^^^ not found in this scope

**Comment by jamesbirtles** [comment]
Looks like this was fixed in #108 but is not yet on crates.io

for now i'm just referencing it via git:
```toml
# Change back to crates.io version when https://github.com/lawliet89/rocket_cors/pull/108 is published
rocket_cors = { git = "https://github.com/lawliet89/rocket_cors.git", rev = "54fae070"  }
```

**Comment by hamidrezakp** [comment]
Why isn't there any new release with this commit in it? it's been a while.

**Comment by mike-at-hypothetic** [comment]
Yes I'm getting this in 0.6.0-alpha1 as well.  The direct link above indeed works, but I eagerly await its appearance on crates.io.
